en/starter/generator.md

```diff
@@ -74,12 +74,18 @@ On MacOS or Linux, run the app with this command:
 $ DEBUG=myapp:* npm start
 ```
 
-On Windows, use this command:
+On Windows (Command Prompt), use this command:
 
 ```sh
 > set DEBUG=myapp:* & npm start
 ```
 
+On Windows (PowerShell), use this command:
+
+```sh
+> set DEBUG=myapp:* ; npm start
```

Comment:
This doesn't seem to work. Here it the above Command Prompt version showing it visible in Node.js:
```
C:\>set DEBUG=myapp:* & node -pe "process.env.DEBUG"
myapp:*
```
Here is the PowerShell one provided showing the variable is not present in Node.js:
```
PS C:\> set DEBUG=myapp:* ; node -pe "process.env.DEBUG"
undefined
```